### PR TITLE
chore: rebuild TVOSExample with RN 0.76

### PR DESCRIPTION
**Pull request by alduzy**
## Description

This PR rebuilds TVOSExample with version 0.76 of `react-native` and without `expo` using the [template](https://github.com/react-native-tvos/react-native-tvos?tab=readme-ov-file#project-creation-using-the-react-native-community-cli).

## Checklist

- [x] Ensured that CI passes

### CODE REVIEW

**Review by kkafar — COMMENTED** [review]
Have you made sure that this actually opens TV os app and not just iOS one? :D 

+ I think we can remove all Android code, right? (separate PR)

File: .github/workflows/tv-os-build-test.yml [hunk 1]
```diff
@@ -40,4 +40,4 @@ jobs:
         run: pod install
       - name: Build app
         working-directory: ${{ env.WORKING_DIRECTORY }}
-        run: npx react-native run-ios --scheme="TVOSExample-tvOS" --simulator="Apple TV"
+        run: npx react-native run-ios --simulator="Apple TV"
```
[the remark this answers is not in the document]
Comment (alduzy) [reply .github/workflows/tv-os-build-test.yml:43]: It's a profile with predefined settings for the project ([docs](https://developer.apple.com/documentation/xcode/customizing-the-build-schemes-for-a-project))

I started this app from the template and it didn't have one so I adjusted the command to use the default scheme.

File: TVOSExample/android/app/build.gradle [hunk 2]
```diff
@@ -109,13 +109,10 @@ android {
 
 dependencies {
     // The version of react-native is set by the React Native Gradle Plugin
-    // For the TV repo,
-    // we use the io.github.react-native-tvos group for the react-android and hermes-android dependencies
-
-    implementation("io.github.react-native-tvos:react-android")
+    implementation("com.facebook.react:react-android")
```
[the remark this answers is not in the document]
Comment (alduzy) [reply TVOSExample/android/app/build.gradle:112]: It's all generated by the [template](https://github.com/react-native-tvos/react-native-tvos?tab=readme-ov-file#project-creation-using-the-react-native-community-cli).

File: TVOSExample/package.json [hunk 3]
```diff
@@ -3,15 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "expo run:android",
-    "ios": "expo run:ios",
-    "web": "expo start --web",
-    "start": "expo start",
-    "postinstall": "patch-package",
-    "clean-ios": "rm -rf ios/build ios/Pods ios/Podfile.lock",
-    "setup-ios": "RCT_NEW_ARCH_ENABLED=1 npx pod-install",
```
[the remark this answers is not in the document]
Comment (alduzy) [reply TVOSExample/package.json:12]: back in https://github.com/software-mansion/react-native-screens/pull/2486/commits/72cec8d92311c7c8513f1d97270b43eeeca7aa9e

**Comment by alduzy** [comment]
@kkafar I tested it on Apple TV simulator and it works just fine. I wouldn't remove the android directory as it's created by default by the template.

**Review by kkafar — APPROVED** [review]
Okie dokie, we're looking good. Thanks!

**Pull request merged**